src/documents/integrationIntro.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Preferences Server Integration
+layout: default
+category: Overview
+---
+
+The First Discovery Tool supports integration with the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to save user selected preferences. The First Discovery Tool has been designed for users who are new to the concept of user interface customization and setting preferences. It is assumed that users will not already have any preferences stored on the GPII Preferences Server and that the First Discovery Tool will walk them through the process of creating their first preference set. This integration will save the selected preferences to the GPII Preferences server and return a token to the user. The token will allow the user to retrieve the saved preferences when needed.
+
+**Note**: While this document focuses on integration with the GPII Preferences Server, the First Discovery Tool could be integrated with any data store to save a user's preferences.
+
+![Diagram showing the integration of the First Discovery Tool and the GPII Preferences Server](images/GPII-Preferences-Server-Integration.svg)
```

Comment:
The image is not in the right place in the source tree and so is not showing up in the output; it needs to be at src/static/images.